analyzer/config/docker/docker.go

```diff
@@ -0,0 +1,65 @@
+package docker
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/open-policy-agent/conftest/parser/docker"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/config"
+	"github.com/aquasecurity/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&dockerConfigAnalyzer{
+		parser: &docker.Parser{},
+	})
+}
+
+const version = 1
+
+var requiredExts = []string{"Dockerfile"}
```

Comment:
We don't have a plan to add more extensions for now. Also, it is not an extension.

```suggestion
var requiredFile = "Dockerfile"
```

Reply:
fixed.